apps/ui/src/composables/useWhiteLabel.ts

```diff
@@ -0,0 +1,75 @@
+import { getNetwork } from '@/networks';
+import { useSpacesStore } from '@/stores/spaces';
+import { NetworkID } from '@/types';
+
+const NETWORK = 's';
+const DEFAULT_DOMAIN = import.meta.env.VITE_HOST || 'localhost';
+const domain = window.location.hostname;
+
+const isWhiteLabel = ref(false);
+const isCustomDomain = ref(domain !== DEFAULT_DOMAIN);
+const failed = ref(false);
+const resolved = ref(domain === DEFAULT_DOMAIN);
+const resolver = reactive<{
+  address: string | null;
+  networkId: NetworkID | null;
+}>({
+  address: null,
+  networkId: null
+});
+
+async function getSpaceId(domain: string): Promise<string | null> {
+  // Resolve white label domain locally if mapping is provided
+  // for easier local testing
+  // e.g. VITE_WHITE_LABEL_MAPPING='127.0.0.1;s:snapshot.eth'
+  const localMapping = import.meta.env.VITE_WHITE_LABEL_MAPPING;
+  if (localMapping) {
+    const [localDomain, localSpaceId] = localMapping.split(';');
+    if (domain === localDomain) return localSpaceId;
+  }
```

Comment:
Im not able to get it working with 

`VITE_WHITE_LABEL_MAPPING='127.0.0.1;s:thanku.eth` 

when I go to http://127.0.0.1:8081/# it still show normal app
maybe I miss some step?

Reply:
What's your full command ?

Reply:
i have added it .env and then `yarn dev`

Reply:
try `VITE_WHITE_LABEL_MAPPING='127.0.0.1;s:snapshot.eth' yarn dev`